doctoc.js

```diff
@@ -17,12 +17,12 @@ function cleanPath(path) {
 }
 
 function transformAndSave(files, mode, maxHeaderLevel, title, notitle, entryPrefix, stdOut) {
-  console.log('\n==================\n');
+  (stdOut) ? console.error('\n==================\n') : console.log('\n==================\n')
```

Comment:
Almost, but ALWAYS log on `console.error` not only if `stdout` is used.
It's the correct thing to do either way when logging (mistake of my past self for not doing so in the first place ;) )